### ISSUE TITLE
Auto completion of "override" in VS editor

### ISSUE THREAD

**Issue by matty-hall** [dotnet/roslyn#9035]
Hi -- I'm not sure if this is the correct repo to log this bug, so please direct me elsewhere if that's not the case!

In VS2013 I often times would type "public override..." and Intellisense would give me a list of _all_ methods to override irrespective of their access (i.e. it wouldn't just show public members).  When I chose a member in the list it would then correct the accessibility from public->protected (or whatever) if I'd chosen wrong.  This was _incredibly_ helpful when overriding methods in a class you're not as familiar with and, thus, didn't know the accessibility of the method you were trying to override.

In VS2015 this behavior has changed.  When I type "public override", the Intellisense (presumably, populated by Roslyn) is filtered to only contain things that match exactly the accessibility you typed.  While I understand the intent here, it's a pretty major regression in functionality, since I now have to go back and change public to protected (or internal) and try again, playing hunt and peck to see what's what.

**Comment by HellBrick** [comment]
You can just type `override` and Intellisense will suggest both public and protected members, with the additional bonus of accessibility modifier being added automatically when you hit `Tab`.

**Comment by matty-hall** [comment]
Ah thanks for the tip.  But as I said, this used to work:

![image](https://cloud.githubusercontent.com/assets/13742340/13243897/0e500a2e-d9b7-11e5-8aa2-0dd68321eebd.png)

Previously, the completion list would include "DoSomething" and when you selected it, the "public" would get corrected to "protected".  I see that "override" works, so thanks for that, but changing stuff like this is sort of frustrating given the insane amount of automatic muscle memory you develop around things like this is you write code for awhile.

**Comment by DustinCampbell** [comment]
Yes, this is a designed behavior change. For VS 2015, we aligned C# and Visual Basic override completion support. C# didn't filter the list based on the accessibility keyword typed, VB did. We decided that the latter behavior was better in general. After all, typing "public" to override a protected member is redundant and wasted typing time. If you don't know the accessibility, typing "override" by itself is best.
